pkg/apis/eventing/v1alpha1/subscription_types.go

```diff
@@ -138,11 +138,18 @@ type SubscriberSpec struct {
 	// +optional
 	Ref *corev1.ObjectReference `json:"ref,omitempty"`
 
+	// Deprecated: Use URI instead.
 	// Reference to a 'known' endpoint where no resolving is done.
 	// http://k8s-service for example
 	// http://myexternalhandler.example.com/foo/bar
 	// +optional
-	DNSName *string `json:"dnsName,omitempty"`
+	DeprecatedDNSName *string `json:"dnsName,omitempty"`
```

Comment:
when will it be removed? the `DeprecatedDNSName` ?

Reply:
#993 tracks removing it. The plan is to leave it deprecated for one release. Then remove it in the next. So if this PR merges into 0.5.0, we would remove it in 0.6.0.